src/main/scala/dsptools/Chisel3TestersCompatibility.scala

```diff
@@ -0,0 +1,14 @@
+package chisel3.iotesters
```

Comment:
Well, if you can make all of the methods I need in this file public, then I won't have an issue. I don't think it's unusual for people to want to override some feature with their own custom thing, so privatizing stuff is annoying. I'm largely adding extra print features, etc. to the tester that maybe people don't want in the base tester... but that requires overriding super class functions, and I don't always have access to the stuff used by the functions b/c I'm not in iotesters. I get that people should maybe not be trying this kind of thing on a regular basis (esp. for hardware writers) but for people trying to extend Chisel features via libraries, the privatizing thing is very limiting. Is there some way around that? And I don't mean just telling me that x,y,z is dangerous and that's why we're not supporting it (I feel like I've heard that way too much this past year) -- b/c I and others will find ways like this to go around stuff.